Cargo.toml

```diff
@@ -3,6 +3,22 @@ name = "rmathlib"
 description = "A Rust Port of R's C Library of Special Functions"
 version = "1.0.0"
 edition = "2021"
+authors = [
```

Comment:
Maybe authors "The PowerAnalyses.org contributors" like https://github.com/rust-random/rand/blob/master/Cargo.toml

Reply:
Following the `libm`'s one: https://github.com/rust-lang/libm/blob/master/Cargo.toml
It is maintained by `rust-lang` with 39 contributors but somehow has a single author.